projects/plugins/jetpack/modules/stats.php

```diff
@@ -339,7 +431,86 @@ function stats_reports_load() {
 	} elseif ( ! isset( $_GET['noheader'] ) && empty( $_GET['nojs'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 		// Normal page load.  Load page content via JS.
 		add_action( 'admin_print_footer_scripts', 'stats_js_load_page_via_ajax' );
+		add_action( 'admin_print_footer_scripts', 'stats_handle_test_button_toggle' );
+		add_action( 'admin_footer', 'stats_reset_nudge_handler' );
+
+		wp_localize_script( 'my-ajax-handle', 'the_ajax_script', array( 'ajaxurl' => admin_url( 'admin-ajax.php' ) ) );
+	}
+}
+
+function stats_reset_nudge_handler() {
+	$nonce = wp_create_nonce();
```

Comment:
```suggestion
	$nonce = wp_create_nonce( 'wp_rest' );
```

Reply:
Thanks! This is the change that got things working.

I'm going to do a bit more research on this and look at any other examples I can find. My reading of the code had me thinking this was for the benefit of the caller (ie: to manage multiple nonces) and not a requirement for the REST auth to work.